.env.example

```
@@ -319,6 +319,15 @@ DEPLOYMENT_ID=
 PROJECT_ID=
 APP_KEY=
 
+# -----------------------------------------------------------------------------
+# Messaging Queue Configuration
+# -----------------------------------------------------------------------------
+MESSAGING_LEASE_DURATION_SECONDS=60
+MESSAGING_MAX_RETRY_ATTEMPTS=5
+MESSAGING_BACKOFF_BASE_SECONDS=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MESSAGING_BACKOFF_BASE_SECONDS key should go before the MESSAGING_LEASE_DURATION_SECONDS key

(UnorderedKey)